.github/workflows/build.yml

```
@@ -48,14 +48,23 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: 3.9.15
-      - name: Auth gcloud
-        uses: google-github-actions/auth@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
```

```suggestion
      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v3
```

For consistency (same everywhere)